knative-operator/pkg/controller/knativekafka/rbacproxytransforms.go

```diff
@@ -0,0 +1,50 @@
+package knativekafka
+
+import (
+	"context"
+	"errors"
+
+	mf "github.com/manifestival/manifestival"
+	operatorv1alpha1 "github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis/operator/v1alpha1"
+	"github.com/openshift-knative/serverless-operator/openshift-knative-operator/pkg/monitoring"
+	"k8s.io/apimachinery/pkg/util/sets"
+	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	kafkaChannelComponents = []string{"kafka-ch-controller", "kafka-ch-dispatcher", "kafka-webhook"}
+	kafkaSourceComponents  = []string{"kafka-controller-manager"}
+)
+
+func addRBACProxySupportToManifest(instance *operatorv1alpha1.KnativeKafka, components []string) (*mf.Manifest, error) {
```

Comment:
not sure I get why this is a bit different than the same for eventing: https://github.com/openshift-knative/serverless-operator/blob/6f9d2ca6d6a8a61b251315d2d326b330094dcdca/openshift-knative-operator/pkg/monitoring/monitoring_eventing.go#L30

Reply:
Because for example there is no need to call `proxyManifest := mf.Manifest{}` and there is a specific rule about `"mt-broker-controller"` sa not all components have the sa name be equal to component name. I could separate the common code but does not add much to do so right now.